src/webhook-handler/validate.js

```diff
@@ -0,0 +1,40 @@
+const AWS = require('aws-sdk/global');
+const SSM = require('aws-sdk/clients/ssm');
+
+const crypto = require('crypto')
+const HTTPError = require('node-http-error');
+
+
+const validate = (message, signature) => {
```

Comment:
**validate** is a generic name - would make more sense being validate request signature or similar.